core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
[Game](https://github.com/libgdx/libgdx/blob/9b8f20b2d0e96e53f0f2b98dc8f6131c810aae71/gdx/src/com/badlogic/gdx/Game.java) has it's own screen. They are similar but 1.2 seconds loading time is a significant improvement. I am merging this code while keeping the Game class as it does provide improvements for loading times and refresh rates. A lot of new content to present